components/prize_pool/wikis/apexlegends/prize_pool_custom.lua

```diff
@@ -40,6 +41,15 @@ function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
 
 	)
 
+	local participantLower = mw.ustring.lower(lpdbData.participant)
```

Comment:
Should it be ustring lower or string lower?

Reply:
Ustring

Reply:
As per discord convo:

https://liquipedia.net/apexlegends/Module:TeamCard/Custom

Should match now (removed the gsub)